src/Databases/DatabaseOnDisk.cpp

```diff
@@ -369,6 +370,20 @@ void DatabaseOnDisk::checkMetadataFilenameAvailability(const String & to_table_n
 void DatabaseOnDisk::checkMetadataFilenameAvailabilityUnlocked(const String & to_table_name) const
 {
     String table_metadata_path = getObjectMetadataPath(to_table_name);
+    String suffix = ".sql.detached";
+    auto max_file_name_length = pathconf("/", _PC_NAME_MAX);
```

Comment:
Why `"/"`? Is the result always the same for all (mounted) filesystems? A comment is required